rules/windows/process_creation/proc_creation_win_bash_file_execution.yml

```diff
@@ -0,0 +1,42 @@
+title: Indirect Command Execution From Script File Via Bash.EXE
+id: 2d22a514-e024-4428-9dba-41505bd63a5b
+related:
+    - id: 5edc2273-c26f-406c-83f3-f4d948e740dd
+      type: similar
+status: experimental
+description: Detects execution of Microsoft bash launcher without any flags to execute the content of a bash script directly. This can be used to potentially bypass defenses and execute linux or windows based binaries directly via bash
```

Comment:
```suggestion
description: Detects execution of Microsoft bash launcher without any flags to execute the content of a bash script directly. This can be used to potentially bypass defenses and execute Linux or Windows-based binaries directly via bash
```